ui/src/sandbox.rs

```diff
@@ -1075,19 +1082,36 @@ mod test {
     }
 
     #[test]
-    fn formatting_code() {
+    fn formatting_code_edition2015() {
         let req = FormatRequest {
-            code: "fn foo () { method_call(); }".to_string(),
+            code: "fn main() { use std::num::ParseIntError; let result: Result<i32, ParseIntError> = try { \"1\".parse::<i32>()? + \"2\".parse::<i32>()? + \"3\".parse::<i32>()? }; assert_eq!(result, Ok(6)); }".to_string(),
+            edition: Some(Edition::Rust2015),
+        };
+
+        let sb = Sandbox::new().expect("Unable to create sandbox");
+        let resp = sb.format(&req).expect("Unable to format code");
+
+        assert!(resp.stderr.contains("error: expected identifier, found `\"1\"`"));
```

Comment:
I found that I cannot pass the original test case but I think this might be passed logically 😓?
Btw, I just found it seems that we didn't run `cargo test` in Travis 👀?

Reply:
> we didn't run `cargo test` in Travis

Yeah, I've been putting off handling that because it's going to require a reasonable amount of work to get CI to handle it (the docker containers need to be able to be passed between build steps without leaking the passwords for uploading them to docker hub)